wire-library/wire-kotlin-serialization/build.gradle

```diff
@@ -38,3 +38,38 @@ kotlin {
     }
   }
 }
+
+afterEvaluate {
+  tasks.create('installLocally') {
+    dependsOn 'publishKotlinMultiplatformPublicationToTestRepository'
+    dependsOn 'publishMetadataPublicationToTestRepository'
+    dependsOn 'publishJvmPublicationToTestRepository'
+    if (kmpJsEnabled) {
+      dependsOn 'publishJsPublicationToTestRepository'
+    }
+  }
+}
+
+// TODO(egorand): Remove when https://github.com/srs/gradle-node-plugin/issues/301 is fixed
+repositories.whenObjectAdded {
+  if (it instanceof IvyArtifactRepository) {
+    metadataSources {
+      artifact()
+    }
+  }
+}
+
+apply from: 'jvm.gradle'
+apply from: "$rootDir/gradle/gradle-mvn-mpp-push.gradle"
+
+publishing {
+  // Use default artifact name for the JVM target
```

Comment:
This was for backwards compatibility in that particular module and can be removed here